crates/bitwarden-exporters/src/models.rs

```diff
@@ -83,7 +82,7 @@ fn from_login(
             .collect(),
         totp: l.totp,
         fido2_credentials: l.fido2_credentials.as_ref().and_then(|_| {
-            let credentials = view.get_fido2_credentials(enc).ok()?;
+            let credentials = view.get_fido2_credentials(&mut key_store.context()).ok()?;
```

Comment:
We call `context` extensively and there are warnings that it is an advanced API.

Reply:
A fair amount of the uses of `context` are to access the raw key material and will dissappear once MasterKey/etc support KeyIds, but yeah we can soften the message a little bit, specially in cases like this where you directly pass the context to a function so a lot of the caveats of context don't apply.

Some of the needed contexts around passkey operations are because they need to decrypt the cipher key to deal with their data, there might be a better way to do it without requiring them to deal with contexts, I've made a ticket in case we can think of something: https://bitwarden.atlassian.net/browse/PM-18113

Reply:
Updated the context comment to remove advanced notice and reorganized it a bit, let me know what you think: 
https://github.com/bitwarden/sdk-internal/pull/8/commits/9c263f996a887781fe5c38998debcd1ef9f6c9c7